paddle/fluid/operators/mkldnn/conv_transpose_mkldnn_op.cc

```diff
@@ -115,10 +115,13 @@ class ConvTransposeMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
     UpdatePaddingAndDilation(&paddings, &dilations, padding_algorithm,
                              data_dims, strides, ksize);
 
-    PADDLE_ENFORCE(
-        dilations.size() == 2 && dilations[0] == 1 && dilations[1] == 1,
-        platform::errors::Unimplemented(
-            "dilation in convolution is not implemented yet"));
+    PADDLE_ENFORCE(dilations.size() == 2,
```

Comment:
PADDLE_ENFORCE should be replaced by PADDLE_ENFORCE_EQ,  please refer to https://github.com/PaddlePaddle/Paddle/wiki/PADDLE_ENFORCE-Rewriting-Specification

Reply:
I later found it is not necessary to check dilations size, because this it already checked input size and guaranteed only conv2d_transpose are implemented

Reply:
Thanks, I have changed it.